src/test/java/org/broadinstitute/consent/http/service/DraftServiceTest.java

```diff
@@ -1,210 +1,78 @@
-        draft.getUUID(), adminUser);
-    assertEquals(adminVisibleDraft.getUUID(), draft.getUUID());
-    assertEquals(adminVisibleDraft.getName(), draft.getName());
-    assertThat(adminVisibleDraft.getStoredFiles(), hasSize(4));
-  }
-
-  @Test
-  void testDeleteDraft() throws Exception {
-    when(gcsService.storeDocument(any(), anyString(), any())).thenReturn(
-        BlobId.of(UUID.randomUUID().toString(), UUID.randomUUID().toString()));
-    User user = createUser();
-    createDraft(user, 3);
-    Set<DraftInterface> loadedDrafts = draftDAO.findDraftsByUserId(
-        user.getUserId());
-    assertThat(loadedDrafts, hasSize(1));
-    draftService.deleteDraft(loadedDrafts.iterator().next(), user);
-    assertThat(draftDAO.findDraftsByUserId(user.getUserId()), hasSize(0));
-  }
+  @Mock
+  private DraftServiceDAO draftServiceDAO;
 
-  @Test
-  void testDeleteDraftsForUser() throws SQLException, IOException {
-    when(gcsService.storeDocument(any(), anyString(), any())).thenReturn(
-        BlobId.of(UUID.randomUUID().toString(), UUID.randomUUID().toString()));
-    User user = createUser();
-    User user2 = createUser();
-    createDraft(user, 3);
-    createDraft(user2, 1);
-    createDraft(user2, 4);
-    assertThat(draftService.findDraftsForUser(user2), hasSize(2));
-    assertThat(draftService.findDraftsForUser(user), hasSize(1));
-    draftService.deleteDraftsByUser(user2);
-    assertThat(draftService.findDraftsForUser(user), hasSize(1));
-    assertThat(draftService.findDraftsForUser(user2), hasSize(0));
-    draftService.deleteDraftsByUser(user2);
-  }
+  @Mock
+  private GCSService gcsService;
 
   @Test
-  void testDeleteAttachmentFromDraft() throws SQLException, IOException {
-    when(gcsService.storeDocument(any(), anyString(), any())).thenReturn(
-        BlobId.of(UUID.randomUUID().toString(), UUID.randomUUID().toString()));
-    User user = createUser();
-    DraftInterface draft = createDraft(user, 3);
-    Set<FileStorageObject> storedFiles = draft.getStoredFiles();
-    assertThat(storedFiles, hasSize(3));
-    for (FileStorageObject file : storedFiles) {
-      draftService.deleteDraftAttachment(draft, user, file.getFileStorageObjectId());
-    }
-    assertThat(draftService.getAuthorizedDraft(draft.getUUID(), user).getStoredFiles(),
-        hasSize(0));
+  public void testCreateDraft() throws SQLException {
+    doThrow(new BadRequestException("Bad Request")).when(draftServiceDAO).insertDraft(any());
+    DraftService draftService = new DraftService(draftServiceDAO, gcsService);
+    assertThrows(BadRequestException.class, ()-> draftService.insertDraft(null));
   }
 
   @Test
-  void testStreamingOutput() throws SQLException, IOException {
-    User user = createUser();
-    DraftInterface draft = createDraft(user, 1);
-    StreamingOutput output = draftService.draftAsJson(draft);
+  public void testStreamingOutput() throws SQLException, IOException {
```

Comment:
I think the `throws` clause here is wrong; the called methods don't throw `SQLException`. For test code, i think it's easier to use `throws Exception` since a test method doesn't care what exceptions are thrown; any will cause the test to fail.
```suggestion
  public void testStreamingOutput() throws Exception {
```